### ISSUE TITLE
react-player__preview missing role attribute

### ISSUE THREAD

**Issue by phil-sola** [cookpete/react-player#1937]
I recently scanned a new staging site I am building which uses react-player for accessibility issues and the react-player__preview element was flagged for having a missing role attribute.

**Interactive elements must have a correct role assigned**

Fix the following (From browserstack accessibility scanner):

Ensure that a widget role is set on the element. Add the appropriate role from the set of ARIA-prescribed widget roles: https://www.w3.org/TR/wai-aria-1.2/#widget_roles. If the element is composed of multiple interactive elements, add a composite role to the element from the set of composite widget roles set by ARIA. If the element is not supposed to be interacted with, add a role="none" or role="presentation" to it.

Is there any chance of getting this added to the preview element to avoid this being flagged? 

Seems like a quick and easy fix to make the site slightly more accessible.
